### PR TITLE
Add Edit.GoToMember default hotkeys

### PR DESCRIPTION
**Pull request by angusgraham**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@angusgraham : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

File: docs/ide/default-keyboard-shortcuts-in-visual-studio.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ The sections in the following table include commands that are global in that you
 |Edit.GoToDeclaration|**Ctrl+F12**|
 |Edit.GoToDefinition|**F12**|
 |Edit.GoToFindCombo|**Ctrl+D**|
+|Edit.GoToMember|**Ctrl+1, Ctrl+M**<br /><br /> or<br /><br /> **Ctrl+1, M**<br /><br /> or<br /><br /> **Ctrl+\\**|
```
Comment (gewarren) [inline docs/ide/default-keyboard-shortcuts-in-visual-studio.md]:
```suggestion
|Edit.GoToMember|**Ctrl+1, Ctrl+M** or **Ctrl+1, M** or **Alt+\\**|
```

**Comment by PRMerger7** [comment]
@angusgraham : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@angusgraham : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Pull request merged**